Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "TestColliderComponent.h"
+
+#include <AZTestShared/Math/MathTestHelpers.h>
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzManipulatorTestFramework/IndirectManipulatorViewportInteraction.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <AzToolsFramework/ToolsComponents/EditorNonUniformScaleComponent.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/Viewport/ViewportSettings.h>
+#include <AzToolsFramework/ViewportSelection/EditorDefaultSelection.h>
+#include <AzToolsFramework/ViewportSelection/EditorInteractionSystemViewportSelectionRequestBus.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/ViewportUi/ViewportUiManager.h>
+#include <EditorColliderComponent.h>
+#include <Tests/Viewport/ViewportUiManagerTests.h>
+
+namespace UnitTest
+{
+    class PhysXColliderComponentModeTest : public ToolsApplicationFixture<false>
```

Comment:
nit: Could you add a quick comment for what the template is for please

Reply:
I didn't make this file, just moved the the class into a header, not too sure what the template is for

Reply:
Hmm okay no worries, I'll have a check later 😝 (it's possible @jonawals might know)